### PR TITLE
Added the password field to the user class and updated it in the datastore

### PR DESCRIPTION
**Pull request by masterford**
… store

### CODE REVIEW

File: src/main/java/codeu/model/store/basic/DefaultDataStore.java [hunk 1]
```diff
@@ -98,7 +98,8 @@ private void addRandomUsers() {
     Collections.shuffle(randomUsernames);
 
     for (int i = 0; i < DEFAULT_USER_COUNT; i++) {
-      User user = new User(UUID.randomUUID(), randomUsernames.get(i), Instant.now());
+      User user = new User(UUID.randomUUID(), randomUsernames.get(i), Instant.now(),
```
Comment (swarelle) [inline src/main/java/codeu/model/store/basic/DefaultDataStore.java]: I think if the parameters can't all fit on one line, it's more readable if you make each parameter on its own line!

Comment (willjschmitt) [reply src/main/java/codeu/model/store/basic/DefaultDataStore.java]: Generally, we don't do this, except in initializing data. I would just keep to 80char width at least, and wrap when it reaches that.

However, the wrapped line should be vertically aligned with the open paren after User or all on the next line.

Example:
```
User user = new User(UUID.randomUUID(), randomUsernames.get(i), Instant.now(),
                     randomUsernames.get(i));
```

or:
```
User user = new User(
    UUID.randomUUID(), randomUsernames.get(i), Instant.now(), randomUsernames.get(i));
```

or if that doesn't fit:
```

or:
```
User user = new User(
    UUID.randomUUID(), randomUsernames.get(i), Instant.now(),
    randomUsernames.get(i));
```

File: src/main/java/codeu/model/data/User.java [hunk 2]
```diff
@@ -30,10 +31,11 @@
    * @param name the username of this User
    * @param creation the creation time of this User
```
Comment (willjschmitt) [inline src/main/java/codeu/model/data/User.java]: Please at the new param to the docstring

File: src/main/java/codeu/model/data/User.java [hunk 3]
```diff
@@ -50,4 +52,8 @@ public String getName() {
   public Instant getCreationTime() {
     return creation;
   }
+
+  public String getPassword () {
```
Comment (willjschmitt) [inline src/main/java/codeu/model/data/User.java]: Please add docstring

**Comment by willjschmitt** [comment]
@swarelle FYI

File: src/main/java/codeu/model/data/User.java [hunk 4]
```diff
@@ -53,6 +54,7 @@ public Instant getCreationTime() {
     return creation;
   }
 
+  /** Returns the passeord of the user */
```
Comment (willjschmitt) [inline src/main/java/codeu/model/data/User.java]: `password` 😄

Comment (masterford) [reply src/main/java/codeu/model/data/User.java]: Haha my bad

Comment (willjschmitt) [inline src/main/java/codeu/model/data/User.java]: Please punctuate comments as prose english (`user.`)

File: src/main/java/codeu/model/data/User.java [hunk 5]
```diff
@@ -30,6 +30,7 @@
    * @param id the ID of this User
    * @param name the username of this User
    * @param creation the creation time of this User
+   * @param password the password of the user
```
Comment (willjschmitt) [inline src/main/java/codeu/model/data/User.java]: `User`

File: src/main/java/codeu/model/data/User.java [hunk 6]
```diff
@@ -53,6 +54,7 @@ public Instant getCreationTime() {
     return creation;
   }
 
+  /** Returns the password of the User */
```
Comment (willjschmitt) [inline src/main/java/codeu/model/data/User.java]: `User.`

Please add the period at the end to complete the sentence

**Pull request merged**